test/benchmark/common/runner.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+print_help() {
+    echo "Usage: $0 bench_name"
+    echo ""
+    echo "The bench_name argument must be one of the directory under `<asterinas>/test/benchmark/`".
+}
+
+BENCH_NAME=$1
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+
+# Validate arguments
+check_directory() {
+    if [ -z "${BENCH_NAME}" ]; then
+        echo "Error: No directory provided."
+        exit 1
+    fi
+
+    local full_path="${SCRIPT_DIR}/../${BENCH_NAME}"
+
+    if ! [ -d "${full_path}" ]; then
+        echo "Directory '${BENCH_NAME}' does not exist in the script directory."
+    fi
+}
+
+check_directory
+
+BENCH_DIR=${SCRIPT_DIR}/../$1/run.sh
+
+# Prepare the environment
+if [ ! -d /tmp ]; then
+    mkdir /tmp
+fi
+/sbin/ldconfig
+chmod +x ${BENCH_DIR}
```

Comment:
This is a confusing name. From the name, it looks like the variable is a dir. Seeing a directory being given `+x` permission and later run as a command is very confusing...